jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -34,6 +35,21 @@
  */
 public class LayerEntry {
 
+  /**
+   * Stringifies {@code path} in unix form.
+   *
+   * @param path the path
+   * @return the string form of the path
+   */
+  private static String toUnixPath(Path path) {
+    StringJoiner pathJoiner =
```

Comment:
Should we just be always using absolute path?

Reply:
Actually resolving this in this PR now.